### PR TITLE
Change Column Editor to Foundation Dropdown GAWB-2343

### PR DESCRIPTION
**Pull request by UrsaStutsman**
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [x] **Product Owner** sign off
- [x] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Review by gdlex4015 — APPROVED** [review]

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Table.scala [hunk 1]
```diff
@@ -78,10 +78,21 @@ case class Table(queryString: QueryString)(implicit webDriver: WebDriver)
...
+      val colToBeHidden = Checkbox(TestId(s"$header-column-toggle"))
       colToBeHidden.ensureUnchecked()
       val action = new Actions(webDriver)
       action.sendKeys(Keys.ESCAPE).perform()
     }
   }
+
+  def moveColumn(header: String, otherHeader: String): Unit = {
+    val allHeaders = readAllText(columnHeaders)
+    if (allHeaders.contains(header) & allHeaders.contains(otherHeader)) {
```
Comment (breilly2) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Table.scala]: This should be `&&`. Maybe `&` works too but definitely not for good reasons.

**Review by breilly2 — APPROVED** [review]
Only one change then 👍 . I didn't scrutinize the cljs too closely but I saw it in action and it looks good.

**Comment by UrsaStutsman** [comment]
passed automation tests here: https://fc-jenkins.dsp-techops.broadinstitute.org/view/FIAB/job/swatomation-pipeline/179/

**Review by MatthewBemis — APPROVED** [review]
I didn't try it out but the changes look reasonable to me :+1:

**Pull request merged**